interfaces/cython/cantera/onedim.pyx

```diff
@@ -837,37 +837,51 @@ cdef class Sim1D:
         nPoints = [len(flow_domains[0].grid), 12, 24, 48]
 
         for N in nPoints:
+            if N > self.get_max_grid_points():
```

Comment:
I think that's probably correct, because turning the methods into a property on `FlameBase` causes the `get` method to require an argument in this location.